pkg/server/chaosd/network.go

```diff
@@ -310,47 +312,53 @@ func (s *Server) applyPortOccupied(attack *core.NetworkCommand) error {
 		return nil
 	}
 
-    err, flag := checkPortIsListened(attack.Port)
-    if err == nil && flag == true  {
+    flag, err := checkPortIsListened(attack.Port)
+    if flag && err == nil{
     	return errors.Errorf("port %s has been occupied", attack.Port)
 	}
 	
 	c := fmt.Sprintf("nc -l %s", attack.Port)
-	cmd1 := exec.Command("sh", "-c", c)
-	err = cmd1.Start()
+	cmd := exec.Command("sh", "-c", c)
+	err = cmd.Start()
```

Comment:
how about using `output, err := cmd.CombinedOutput()`? so we can print output to know the detail when cmd run failed